src/chain/network_event.rs

```diff
@@ -55,11 +57,19 @@ pub struct SendAckMessagePayload {
     pub ack_version: u64,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
+pub struct SectionInfoSigPayload {
+    /// The signature share signing the SectionInfo.
+    pub share: (BlsPublicKeyShare, BlsSignatureShare),
+    /// The key set to combine shares.
+    pub pk_set: BlsPublicKeySet,
+}
+
 /// Routing Network events
 // TODO: Box `SectionInfo`?
 #[allow(clippy::large_enum_variant)]
 #[derive(Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
-pub enum NetworkEvent {
+pub enum AccumulatingEvent {
```

Comment:
I'd try to think about different name for this. The problem is that there is nothing in the name `AccumulatingEvent` that would suggest it is tied to `NetworkEvent`.Maybe `NetworkEventPayload` would be a better name? Not sure.

Anyway, this is a minor point and something that can be addressed later (or never), so I won't block the PR over it.

Reply:
Yes, I was puzzling the name out, and I liked that this name point out it is the part we are accumulating, can rename later if it is causing confusion.

Reply:
OK, let's not worry about this right now. We can always address it in the future.